subsys/bluetooth/audio/ots.h

```diff
@@ -0,0 +1,322 @@
+/* @file
+ * @brief Object Transfer
+ *
+ * For use with the Object Transfer Service (OTS)
+ *
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_HOST_AUDIO_OTS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_HOST_AUDIO_OTS_H_
```

Comment:
Oh, I was under the impression that we actually had started using the upstreamed OTS implementation, but I see that the otc implementation still uses this. If we don't remove the otc_internal, suggest to move this code into that for now, with a TODO.